Sources/PixelEditor/Components/StepSlider.swift

```diff
@@ -36,6 +36,11 @@ public final class StepSlider : UIControl {
       setupValues()
     }
   }
+  public var displayNumericValue = true {
+    didSet {
+      internalSlider.stepLabel.isHidden = !displayNumericValue
+    }
+  }
```

Comment:
I'd like to make it like this.

- Align naming
- Use computed property (avoid to manage the flag as several places. `internalSlider.stepLabel` be a source of truth)

```suggestion
  public var isStepLabelHidden: Bool {
    get { internalSlider.stepLabel.isHidden }
    set { internalSlider.stepLabel.isHidden = newValue }
  }
```

And, this suggestion will cause compile-error on caller.